<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryDlg</class>
 <widget class="QDialog" name="QueryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>query</string>
  </property>
  <widget class="QDateTimeEdit" name="dte_Begin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>194</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dte_End">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>194</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>681</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="tv_EventInfo">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>20</y>
       <width>561</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>111</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="btn_query">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>query</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
